seatsio-ios/SeatsioObject.swift

```diff
@@ -32,4 +37,81 @@ public struct SeatsioObject: Decodable {
     public let numberOfSelectedObjects: Int?
     public let selectableCategories: [Category]?
     public let isInteractive: Bool?
+
+    public init(from decoder: Decoder) throws {
```

Comment:
do we need this?

Reply:
yes, I encountered a problem with `extraData`. It's of type `Data?` and it triggered type mismatch error, because in javascript it's an arbitrary payload, we don't know what values to expect in array here